addons/cis-hardening/enable

```diff
@@ -85,6 +86,8 @@ def create_certificate(name, group):
             cmd += f"/O={group}"
         subprocess.check_call(cmd.split(), stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
         cmd = f"{snap_path}/usr/bin/openssl x509 -req -in {csr} -CA {cacert} -CAkey {cakey} -CAcreateserial -out {cert} -days 3650"
+        if extfile:
+            cmd += f"-extfile {extfile}"
```

Comment:
I'm not sure if we might need a space here, without it would be `... -days 3650-extfile {extfile}` which I'm not sure if the argument would register properly.
```suggestion
            cmd += f" -extfile {extfile}"
```